### PR TITLE
[Feature] Add laravel 6 support

### PR DESCRIPTION
**Pull request by JaskirtPooni**
Had to update a few tests as well.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
     ],
     "require": {
         "guzzlehttp/guzzle": "^6.3",
-        "illuminate/support": "^5.4",
-        "phpunit/phpunit": "~6.0|~7.0|~8.0"
+        "illuminate/support": "^6.0",
```
Comment (jshayes) [inline composer.json:13]: I'm down with not supporting 5.8 anymore since LTS will end for it soon.

File: .travis.yml [hunk 2]
```diff
@@ -2,10 +2,6 @@ language: php
 
 matrix:
   include:
-    - php: 7.1
-      env: ILLUMINATE_VERSION=5.7.*
-    - php: 7.1
-      env: ILLUMINATE_VERSION=5.8.*
     - php: 7.2
       env: ILLUMINATE_VERSION=5.7.*
```
Comment (jshayes) [inline .travis.yml]: Can get rid of the matrix I think. All that is needed for now would be 
```
php:
  - '7.2'
  - '7.3'
  - '7.4'
```

**Review by jshayes — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (JaskirtPooni) [reply composer.json:13]: I don't think it would be possible to support both together either

**Review by jshayes — APPROVED** [review]

**Comment by jshayes** [comment]
This is available in release version 3.0

**Comment by JaskirtPooni** [comment]
Thanks for the super quick review @jshayes

**Pull request merged**